pkg/networkmanager/network-interface.jsx

```diff
@@ -264,7 +264,7 @@ export const NetworkInterfacePage = ({
         return (
             <DescriptionListGroup>
                 <DescriptionListTerm>{_("Status")}</DescriptionListTerm>
-                <DescriptionListDescription>
+                <DescriptionListDescription className="networking-interface-status">
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19023-20230626-214120-185a4bfd-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)